concepticondata/conceptlists/KBA-2021-912.tsv

```diff
@@ -0,0 +1,913 @@
+KBA-2021-912-761	761	m591-k-bp-fl-fa	pain (of body)	1783	PAIN		douleur	Schmerz	pyn, smart	botlhoko		8. Sensations and Feelings	M544		
+KBA-2021-912-762	762	m592-k-bp-fl-fa	to know	3626	KNOW		connaître, savoir	kennen, wissen	ken, weet	go itse		8. Sensations and Feelings	M553	s083	lj058
+KBA-2021-912-763	763	m593-k-bp-fl-fa	to learn	504	LEARN		apprendre	lernen	leer, studeer	go ithuta		8. Sensations and Feelings	M157		
+KBA-2021-912-764	764	m594-k-bp-fl-fa	to remember	707	REMEMBER		se souvenir	sich erinnern	herinner	go gakologelwa		8. Sensations and Feelings	M550		
+KBA-2021-912-765	765	m595-k-bp-fl-fa	to forget	1523	FORGET		oublier	vergessen	vergeet	go lebala		8. Sensations and Feelings	M551		
+KBA-2021-912-766	766	m596-k-bp-fl-fa	to think (reason, reflect)	1415	THINK (REFLECT)		penser, raisonner, réfléchir	denken, nachdenken	dink, nadink	go akanya		8. Sensations and Feelings	M552	s166	
+KBA-2021-912-767	767	m597-k-bp-fl-fa	to think (believe, be of the opinion)	1513	THINK (BELIEVE)		penser, croire	denken, glauben, meinen	dink, glo, meen			8. Sensations and Feelings			
+KBA-2021-912-768	768	m598-k-bp80-fl-fa	to see (experience)	1409	SEE		voir	sehen	sien	go bona		8. Sensations and Feelings	M556	s130	lj089
+KBA-2021-912-769	769	m599-k-bp-fl-fa	to look at (activity)	1819	LOOK		regarder	ansehen, betrachten, anschauen	aankyk	go leba		8. Sensations and Feelings	M154		
+KBA-2021-912-770	770	m6-k99-bp-fl-fa	girl	1646	GIRL		fille	Mädchen	meisie	morweetsana, ba-; mosetsana, ba-		1. Social life  1A.Social structure	M127		
+KBA-2021-912-771	771	m60-k-bp-fl-fa	quarrel	1848	QUARREL		lutte, querelle	Streit, Zank	stryd, twis	dikomana,komano		1C. Social and Antisocial Behaviour	M160		
+KBA-2021-912-772	772	m600-k-bp-fl-fa	to look   (phenomenon)	1819	LOOK		apparaître, ressembler	aussehen				8. Sensations and Feelings			
+KBA-2021-912-773	773	m601-k-bp82-fl-fa	to hear  (experience)	1408	HEAR		entendre	hören	hoor	go utlwa		8. Sensations and Feelings	M554	s069	lj061
+KBA-2021-912-774	774	m602-k-bp83-fl-fa	to listen to (activity)	1608	LISTEN		écouter	anhören, sich anhören, zuhören	luister			8. Sensations and Feelings			
+KBA-2021-912-775	775	m603-k-bp-fl-fa	to sound (phenomenon)												
+KBA-2021-912-776	776	m604-k-bp-fl-fa	sound	1811	SOUND OR NOISE		son	Geräusch	geluid, klank	modumo ('noise'?)		8. Sensations and Feelings	M555		
+KBA-2021-912-777	777	m605-k-bp-fl-fa	noise	1182	NOISE		bruit	Lärm	geraas, lawaai			8. Sensations and Feelings			
+KBA-2021-912-778	778	m606-k-bp-fl-fa	to feel (experience)	1079	FEEL		sentir	fühlen	voel	go utlwa		8. Sensations and Feelings	M557		
+KBA-2021-912-779	779	m607-k-bp-fl-fa	to touch, feel (activity)	2559	FEEL (TACTUALLY)		toucher	anfassen, befühlen, betasten	aanraak	go tshwara		8. Sensations and Feelings	M557		
+KBA-2021-912-780	780	m608-k-bp-fl-fa	to feel (phenomenon)												
```

Comment:
1079 FEEL ? Similar to SMELL below.

Reply:
Not sure what is meant by (phenomenon) but judging by the mapping of 'to smell (phenomenon)' to SMELL, it might be an umbrella concept. But I would unmap to be on the safe side unless there is a way to disambiguate it.